File: molecular/src/test/java/com/hartwig/actin/molecular/orange/util/AminoAcidTest.java
package com.hartwig.actin.molecular.orange.util;

import static org.junit.Assert.assertEquals;

import org.junit.Test;

public class AminoAcidTest {

    @Test
    public void canConvertAminoAcidsToSingleLetter() {
        assertEquals("p.V600E", AminoAcid.forceSingleLetterAminoAcids("p.Val600Glu"));
        assertEquals("p.V600E", AminoAcid.forceSingleLetterAminoAcids("p.V600E"));
    }
}

File: molecular/src/test/java/com/hartwig/actin/molecular/orange/interpretation/AminoAcidTest.kt
package com.hartwig.actin.molecular.orange.interpretation

import org.junit.Assert
import org.junit.Test

class AminoAcidTest {
    @Test
    fun canConvertAminoAcidsToSingleLetter() {
        Assert.assertEquals("p.V600E", AminoAcid.forceSingleLetterAminoAcids("p.Val600Glu"))
        Assert.assertEquals("p.V600E", AminoAcid.forceSingleLetterAminoAcids("p.V600E"))
    }
}